scripts/intfstat

```diff
@@ -17,7 +17,7 @@ import time
 
 # mock the redis for unit test purposes #
 try:
-    if os.environ["UTILITIES_UNIT_TESTING"] == "1":
+    if os.environ["UTILITIES_UNIT_TESTING"] == "2":
```

Comment:
What does "2" represent, and how does it differ from "1"?

Reply:
1 means skip running the command, 2 means you still run the command, but inside the command, you do db mock. I can add some doc in the next pr to document this behavior.

Reply:
> I can add some doc in the next pr to document this behavior.

Please do.